torch/_functorch/eager_transforms.py

```diff
@@ -492,9 +495,23 @@ def compute_jacobian_stacked():
             for flat_basis_chunk in _chunked_standard_basis_for_(flat_output,
                                                                  flat_output_numels,
                                                                  chunk_size=chunk_size):
+                if chunk_size == 1:
+                    flat_basis_chunk = tree_map(lambda t: torch.squeeze(t, 0), flat_basis_chunk)
```

Comment:
should this be tree_map_only? (what happens for non-Tensor arguments?)

Also, squeeze is a bit dangerous, because if you squeeze on a dimension of size 2 it doesn't actually do anything. Would prefer an assertion that t.shape(0) == 1

Reply:
These `flat_basis_chunk` are guaranteed to be `tensors`. Will add an assert.

Ref: https://github.com/pytorch/pytorch/blob/f62a3cabfc53569aef37d35ee34de99ec29dc5d7/torch/_functorch/eager_transforms.py#L615